cmd/neofs-node/config.go

```diff
@@ -36,6 +36,7 @@ import (
 	"github.com/nspcc-dev/neofs-node/misc"
 	"github.com/nspcc-dev/neofs-node/pkg/core/container"
 	netmapCore "github.com/nspcc-dev/neofs-node/pkg/core/netmap"
+	"github.com/nspcc-dev/neofs-node/pkg/innerring"
```

Comment:
This (SN importing IR) was always considered harmful. Can we have it in `pkg/network/cache`? SN uses it and `pkg/innerring` imports it as well.

Reply:
Moved the constant